Casks/1/1password-cli.rb

```diff
@@ -1,11 +1,14 @@
 cask "1password-cli" do
   arch arm: "arm64", intel: "amd64"
 
+  os macos: "darwin", linux: "linux"
+
   version "2.30.3"
-  sha256 arm:   "0472133a09a0596119543c0ee7def0b3c08646354432516a18dbfe831d536c88",
-         intel: "79fa39079b038bfefa8dafbc9e6cd1540ad869532ddbb6cf0a0b829edc7f4d48"
+  sha256 arm:          "0472133a09a0596119543c0ee7def0b3c08646354432516a18dbfe831d536c88",
+         x86_64:       "79fa39079b038bfefa8dafbc9e6cd1540ad869532ddbb6cf0a0b829edc7f4d48"
```

Comment:
```suggestion
         x86_64:       "79fa39079b038bfefa8dafbc9e6cd1540ad869532ddbb6cf0a0b829edc7f4d48",
```

Not sure if it's relevant yet, but it seems like it's missing a comma here.

Reply:
Ah yeah, it's also the wrong checksum